Game/Shaders/phong_pbr_fragment.glsl

```diff
@@ -39,127 +36,118 @@ struct PointLight
 	float kq;
 };
 
-struct SpotLight
-{
+struct SpotLight {
 	vec3 pos;
 	vec3 direction;
 	vec3 color;
 	float intensity;
 	float kc;
 	float kl;
 	float kq;
-	float inner_angle;
-	float outer_angle;
+	float innerAngle;
+	float outerAngle;
 };
 
-struct Light
-{
+struct Light {
 	AmbientLight ambient;
 	DirLight directional;
 	PointLight points[8];
-	int num_points;
+	int numPoints;
 	SpotLight spots[8];
-	int num_spots;
+	int numSpots;
 };
 
 uniform Light light;
-uniform vec3 view_pos;
+uniform vec3 viewPos;
 
-void main()
-{    
-	vec3 frag_n = normalize(frag_normal);
-	vec3 view_n = normalize(view_pos - frag_pos);
+void main() {    
+	vec3 fragN = normalize(fragNormal);
+	vec3 viewN = normalize(viewPos - fragPos);
 	
 	// diffuse 
-	vec3 diffuse_color = has_diffuse_map * pow(texture(diffuse_map, vec2(uv.x, 1 - uv.y)).rgb, vec3(2.2)) + (1 - has_diffuse_map) * diffuse_color;
+	vec3 diffuseColor = hasDiffuseMap * pow(texture(diffuseMap, vec2(uv.x, 1 - uv.y)).rgb, vec3(2.2)) + (1 - hasDiffuseMap) * diffuseColor;

```

Comment:
Y-axis flipped twice? In the [texture loader](https://github.com/TBD-org/TBD-Engine/blob/4d9a7a9d7cfeedcf00533581817745465532f508/Project/Source/FileSystem/TextureImporter.cpp#L45) via devIL, the texture is flipped.

Reply:
This is because we have the textures flipped. This is a separate issue. I will add it to issues.

Reply:
Perfect.